ddtrace/contrib/aiohttp/middlewares.py

```diff
@@ -77,11 +101,17 @@ def on_prepare(request, response):
     request_span.finish()
 
 
-def trace_app(app, tracer, service='aiohttp-web'):
+def trace_app(app, tracer, service='aiohttp-web', enable_distributed=False):
```

Comment:
@palazzem should we remove the tracer from here too?  Would not be backwards compatible though